docs/make.bat

```diff
@@ -0,0 +1,50 @@
+@ECHO OFF
+
+REM Copyright 2018 JanusGraph Python Authors
+
+REM Licensed under the Apache License, Version 2.0 (the "License");
+REM you may not use this file except in compliance with the License.
+REM You may obtain a copy of the License at
+
+    REM http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Please put the `REM` comments flush with the left margin, and put the spaces before it (on this line) after it so that it aligns the URL 4 spaces from the margin of the rest of the text.

The license comment block should probably also have `REM` markers on the blank lines between the text as well. This is what I mean: https://github.com/mbrukman/autogen/blob/master/tests/testdata/bat.out (capitalization of `REM` vs. `rem` is up to you). 

Shameless plug: I wrote https://github.com/mbrukman/autogen/ to help generate license header blocks just like this one. :-)

Reply:
Ah thanks for the info @mbrukman . I haven't created much windows batch files so don't know much. 
What do you mean by `comments flush` couldn't understand it 😅 
As for the github repo you shared, I couldn't make use of it, but rather copied the liscence from `bat.out` as I think that's sample one and used the same in my batch file and in other windows batch files as well 😅